src/index.js

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable import/no-cycle */
 /* eslint-disable no-restricted-globals */
 /* eslint-disable import/prefer-default-export */
 import './style.css';
 import { getTask } from './modules/data.js';
-import { deleteTask, deleteOne } from './modules/controllTools.js';
+import { deleteTask, deleteOne, updateTask } from './modules/controllTools.js';
+import { dragDrop } from './modules/dragAndDrop.js';
```

Comment:
## [OPTIONAL]
- [ ] kindly you need to solve your linters instead of disabling them. Consider this in your next projects.